### DISCUSSION TITLE
Can we receive notification when someone request a movie?

### DISCUSSION THREAD

**Discussion by JeremieBergeron** [sct/overseerr#1986] [Q&A] [answered]
Hi,
Is it possible to receive a notification when someone request a movie?
Thank you

**Comment by danshilm** [comment, marked as answer]
Overseerr offers several ways to receive notifications. Just head to Settings -> Notifications and enable the ones you want. The current choices are: Email, Web Push Notifications, Discord, LunaSea, Pushbullet, Pushover, Slack, Telegram and Custom Webhooks.
The "Notification Type" to enable to receive notifications when someone requests content is "Media Requested".
